providers.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
```

Comment:
Is there a way to avoid this call?
If I make this a module and this priovider block is in here then I will not be able to make use of the `count` or `for_each` attributes

Reply:
I deleted this file so I can define the provider at the root level and pass it to the modules. 
in this way I can use count or for_each